include/st_pipeline_api.h

```diff
@@ -156,6 +163,13 @@ enum st_frame_fmt {
   ST_FRAME_FMT_JPEGXS_CODESTREAM = 24,
   /** ST22 h264 cbr codestream */
   ST_FRAME_FMT_H264_CBR_CODESTREAM,
+  /** User defined format.
+   * This is used when user wants to directly transport none-RFC4175 formats like
+   * YUY2/I420/NV12. When this input/output format is set, the frame is identical to
+   * transport frame without conversion. User should take care of the frame size and
+   * planes info.
+   */
+  ST_FRAME_FMT_ANY,
```

Comment:
Maybe we can define ST20_FMT_CUSTOMIZE with a customized pg group to  support non-rfc4175 format

Reply:
This can be added to plan if we want to use st20 to transport non-rfc4175 formats like v210.